index.html

```diff
@@ -29,6 +29,8 @@ <h2>Guides in other languages</h2>
   </div>
 </div>
 
+<button onclick="topFunction()" id="myBtn" title="Go to top"><img class="arrow-img" src="arrow.png" alt="" srcset=""></button>
```

Comment:
```suggestion
<button onclick="topFunction()" id="go-to-top-arrow" title="Go to top"><img class="arrow-img" src="arrow.png" alt="" srcset=""></button>
```

This should have a more descriptive name than `myBtn`.

Reply:
Changes Done !